##
#Constants defining the penalty field

uint16 NO_PENALTY = 0
uint16 BALL_MANPULATION = 1
uint16 PHYSICAL_CONTACT = 2
uint16 ILL_ATTACK = 3
uint16 ILL_DEFENSE = 4
uint16 PICKUP_REQUEST = 5
uint16 SERVICE_REQUEST = 6
uint16 PICK_AND_SERV_REQUEST = 7


uint16 penalty
uint16 remainingPenaltySec
